### PR TITLE
fix: UI adjustments to post summary and post details

### PR DESCRIPTION
**Pull request by A-ASAD**
## Ticket: [TNL-9751](https://openedx.atlassian.net/browse/TNL-9751)

Updated UI of post summary and post details:
Font-weight of summary title
Author label color (in both summary and post details)
Border around avatar/profile image (in both summary and post details)
Spacing between avatar and other details
Spacing between the summary footer and content
Enlarge background of like button

Screenshot 2022-03-31 at 5 56 49 PM

### CODE REVIEW

File: src/data/constants.js [hunk 1]
```diff
@@ -68,6 +68,16 @@ export const RequestStatus = {
...
+/**
+ * Enum for author label and avatar border color classes.
+ * @readonly
+ * @enum {string}
+ */
+export const ColorClasses = {
+  Staff: 'warning-700',
+  'Community TA': 'success-700',
+};
+
```
Comment (awais-ansari) [inline src/data/constants.js:80]: nit: Assign more meaningful name.

**Review by awais-ansari — APPROVED** [review]

File: src/data/constants.js (hunk 1 above)
Comment (A-ASAD) [reply src/data/constants.js:80]: Okay

Comment (mehaknasir) [reply src/data/constants.js:80]: avatarColorClasses?

**Review by mehaknasir — APPROVED** [review]
Code changes looks fine to me. I would like to test on stage. let me know once it is shipped there.

**Pull request merged**